package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "expression-atlas-my-package",
-  "version": "0.0.1",
+  "name": "scxa-homepage-panels",
+  "version": "0.0.3",
   "description": "My package does some neat things",
   "main": "lib/index.js",
   "scripts": {
```

Comment:
Use the `scripts` from the lates version of [our template package](https://github.com/ebi-gene-expression-group/atlas-package/blob/master/package.json).